src/components/App.js

```diff
@@ -9,13 +9,33 @@ class App extends Component {
   constructor() {
       super()
       this.state = {
-        movies: movieData.movies,
-        currentMovie: {}
+        movies: {},
```

Comment:
Is movies a collection of movie objects? In that case, I would initialize this piece of state as an empty array because when the data comes in, then it will fill that empty array. That's less confusing for other developers.

Otherwise I would expect this piece of data to stay a single object.